st2api/st2api/controllers/v1/actions.py

```diff
@@ -78,8 +78,13 @@ def __init__(self, *args, **kwargs):
 
     @request_user_has_permission(permission_type=PermissionType.ACTION_LIST)
     @jsexpose()
-    def get_all(self, **kwargs):
-        return super(ActionsController, self)._get_all(**kwargs)
+    def get_all(self, exclude_attributes=None, **kwargs):
+        if exclude_attributes:
```

Comment:
I like .only() over .exclude() but when we expose this to an user not from stackstorm team, neither approach would make sense without knowing the schema for the object itself which needs to be solved in first place.